examples/cra-demo/src/CardExample.tsx

```diff
@@ -9,31 +9,30 @@ type CardExampleProps = {
 const CardExample = ({ title, cardNumber }: CardExampleProps) => {
   const [isCopied, setCopied] = useState(false);
 
+  function handleCopyButtonClick(): void {
+    setCopied(true);
+    copy(cardNumber);
+    setTimeout(() => {
+      setCopied(false);
+    }, 1000);
+  }
+
   return (
     <li>
       {title}
-      <CopyToClipboard
-        text={cardNumber}
-        onCopy={() => {
-          setCopied(true);
-          setTimeout(() => {
-            setCopied(false);
-          }, 1000);
-        }}
+      <svg
+        aria-hidden="true"
+        onClick={handleCopyButtonClick}
```

Comment:
It's better to wrap this in a `button` rather than adding `onClick` to the SVG element, but [our current CSS target `button` tag](https://github.com/formspree/formspree-js/blob/d00a2815e9ded65eebbc6c9c26d401be0f0fbe65/examples/cra-demo/src/index.css#L66), so we'll fix that in the future.